src/test/java/linenux/command/RemindCommandTest.java

```diff
@@ -0,0 +1,256 @@
+package linenux.command;
+
+import static linenux.helpers.Assert.assertChangeBy;
+import static linenux.helpers.Assert.assertNoChange;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import linenux.command.parser.ReminderArgumentParser;
+import linenux.command.result.CommandResult;
+import linenux.model.Reminder;
+import linenux.model.Schedule;
+import linenux.model.Task;
+
+/**
+ * JUnit test for remind command.
+ */
+public class RemindCommandTest {
+	private Schedule schedule;
+	private RemindCommand remindCommand;
+
+	@Before
+	public void setupRemindCommand() {
```

Comment:
I am not sure there is a need to test for deadline, todo and event separately unless there is some functionality difference (Like cannot add reminder after deadline and event end time). Else I think just testing for a generic task should be fine